index.html

```diff
@@ -35,23 +39,243 @@ <h1>I’m Giorgi Bokuchava glad to see you!</h1>
                 <div class="lets-connect">
                     <h5>LET'S CONNECT</h5>
                 </div>
-
                 <div class="icons">
                     <ul>
                         <li><a href="https://twitter.com"><img src="Pictures/twitter.png" alt=""></a></li>
-                        <li><a href="https://www.linkedin.com/in/giorgi-bokuchava-430252240/"><img src="Pictures/Linkedin.png" alt=""></a></li>
+                        <li><a href="https://www.linkedin.com/in/giorgi-bokuchava-430252240/"><img
+                                    src="Pictures/Linkedin.png" alt=""></a></li>
                         <li><a href="https://medium.com"><img src="Pictures/medium.png" alt=""></a></li>
                         <li><a href="https://github.com/Boku52500"><img src="Pictures/github.png" alt=""></a></li>
                         <li><a href="https://angellist.com"><img src="Pictures/victory.png" alt=""></a></li>
                     </ul>
                 </div>
             </div>
-        </section>
+    </section>
+
+
+
+
+    <section class="2-page">
+        <div class="snapshot">
+            <a href=""><img src="Pictures/snapshot.png" alt=""></a>
+        </div>
+        <h2>Tonic</h2>
+        <div class="bullets">
+            <h3>CANOPY</h3>
+            <ul class="bullets1">
+                <li>Back End Dev</li>
+                <li>2015</li>
+            </ul>
+        </div>
+        <div class="second-para">
+            <h4>A daily selection of privately personalized reads; no accounts or sign-ups required.</h4>
+        </div>
+        <div class="alltext">
+            <div class="text1">
+                html
+            </div>
+            <div class="text2">
+                css
+            </div>
+            <div class="text3">
+                javaScript
+            </div>
+        </div>
```

Comment:
- [ ] Kindly make this section as a list `ul` where each item `li` is one of the technologies 👍